spikeextractors/extractorlist.py

```diff
@@ -34,6 +34,7 @@
 from .extractors.alfsortingextractor import ALFSortingExtractor
 from .extractors.cedextractors import CEDRecordingExtractor
 from .extractors.cellexplorersortingextractor import CellExplorerSortingExtractor
+from .extractors.neuropixelsrecordingextractor import NeuropixelsRecordingExtractor
```

Comment:
@mhhennig still note convinced about the name..what about `NeuropixelsDatRecordingExtractor`? At least this makes it clear it needs a dat file